packages/components/src/segmented-control/segmented-control-button.tsx

```diff
@@ -0,0 +1,70 @@
+/**
+ * External dependencies
+ */
+import { cx } from 'emotion';
+// eslint-disable-next-line no-restricted-imports
+import { Radio } from 'reakit';
+
+/**
+ * WordPress dependencies
+ */
+import { memo } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import {
+	ButtonContentView,
+	LabelPlaceholderView,
+	LabelView,
+	SeparatorView,
+} from './styles';
+// eslint-disable-next-line no-duplicate-imports
+import * as styles from './styles';
```

Comment:
To avoid this duplicate import we could just destructure the first imports from the `* as styles` import
```suggestion
import * as styles from './styles';

const {
	ButtonContentView,
	LabelPlaceholderView,
	LabelView,
	SeparatorView,
} = styles;
```